api/axios.ts

```
@@ -60,7 +60,9 @@ export const axiosInstance = axios.create({
  * - TODO: 추후 SecureStore에서 토큰 가져오도록 수정 필요
  */
 axiosInstance.interceptors.request.use(async (config) => {
+  const token = await getClerkToken();
   const token = await getClerkToken();
+  if (token) {
   if (token) {
     config.headers.Authorization = `Bearer ${token}`;
```

Biome (2.1.2)

[error] Shouldn't redeclare 'token'. Consider to delete it or rename it.

'token' is defined here:


(lint/suspicious/noRedeclare)